meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx

```diff
@@ -6,42 +6,46 @@ import { ListItemBadges } from './ListItemBadges'
 import { ListItemStats } from './ListItemStats'
 
 export const BudgetingProposalListItem = (props) => {
-  const { proposal, isVotingPhase } = props
-  const safeLocale = proposal.locale ? proposal.locale : undefined
+  const { proposal: p, isVotingPhase } = props
+  const safeLocale = p.locale ? p.locale : undefined
```

Comment:
I don't understand what you're doing here, why change it to p?

Reply:
just to keep the following lines (12 and 13) in one line.
think with `p` it is still readable or even more readable, no?

Reply:
i could also do it with, so we can get rid of the `p` entirely.

```
  const  {
    name,
    locale,
    ... } = proposal

Reply:
Thanks for adding proposals back! I was mostly weirded out with the reassignment in const, I would come back in a month and wonder why it's there :p